### DISCUSSION TITLE
Possible issue with copying from text box

### DISCUSSION THREAD

**Discussion by rRobis** [SubtitleEdit/subtitleedit#8326] [General]
I've noticed for the past couple months, that sometimes when I copy text from text box, it doesn't copy (neither by CTRL+C, neither by context menu). Then I try to copy another text in the same text box, and it then gets unstuck and I can copy again.

I'm not sure if this is a problem with my setup or not, but SE is the only place where I noticed this happening.

I tried to disable AutoHotkey, and it seems that it's not the cause of the problem (since it also didn't copy with context menu copy button). 

I think it started to happen, when the way the selection with mouse was changed in text box (double click, drag to select more words etc, if i'm not mistaken). I can't notice the pattern yet, so i'm clueless at this point, what could cause this.

Has anyone noticed something similar?

**Comment by GrampaWildWilly** [comment]
I just did a quick test using 4.0.5 beta 159 on Windows 11.  It copied the text of one subtitle from the text box to the clipboard as expected.  I then pasted it in here no problem.  (And then I deleted it from here since it's just noise at this point.)  But I use Ctrl+Insert to copy & Shift+Insert to paste.  Maybe that makes a difference.

**Reply by rRobis** [reply]
It happens randomly for me. If i were to do it now, it would probably work, but sometimes it just gets stuck for some reason.
